src/gui/MixerView.cpp

```diff
@@ -68,123 +68,116 @@ MixerView::MixerView() :
+	chLayout->setSizeConstraint(QLayout::SetMinimumSize);
+	chLayout->setSpacing(0);
 	chLayout->setContentsMargins(0, 0, 0, 0);
 	m_channelAreaWidget->setLayout(chLayout);
 
 	// create rack layout before creating the first channel
 	m_racksWidget = new QWidget;
-	m_racksLayout = new QStackedLayout( m_racksWidget );
-	m_racksLayout->setContentsMargins( 0, 0, 0, 0 );
-	m_racksWidget->setLayout( m_racksLayout );
+	m_racksLayout = new QStackedLayout(m_racksWidget);
+	m_racksLayout->setContentsMargins(0, 0, 0, 0);
+	m_racksWidget->setLayout(m_racksLayout);
 
 	// add master channel
-	m_mixerChannelViews.resize( m->numChannels() );
-	m_mixerChannelViews[0] = new MixerChannelView( this, this, 0 );
+	m_mixerChannelViews.resize(m->numChannels());
+	m_mixerChannelViews[0] = new MixerChannelView(this, this, 0);
 
-	m_racksLayout->addWidget( m_mixerChannelViews[0]->m_rackView );
+	m_racksLayout->addWidget(m_mixerChannelViews[0]->m_effectRackView);
 
 	MixerChannelView * masterView = m_mixerChannelViews[0];
-	ml->addWidget( masterView->m_mixerLine, 0, Qt::AlignTop );
+	ml->addWidget(masterView, 0, Qt::AlignTop);
 
-	QSize mixerLineSize = masterView->m_mixerLine->size();
+	auto mixerChannelSize = masterView->sizeHint();
 
 	// add mixer channels
-	for( int i = 1; i < m_mixerChannelViews.size(); ++i )
+	for (int i = 1; i < m_mixerChannelViews.size(); ++i)
 	{
 		m_mixerChannelViews[i] = new MixerChannelView(m_channelAreaWidget, this, i);
-		chLayout->addWidget( m_mixerChannelViews[i]->m_mixerLine );
+		chLayout->addWidget(m_mixerChannelViews[i]);
 	}
 
 	// add the scrolling section to the main layout
 	// class solely for scroll area to pass key presses down
 	class ChannelArea : public QScrollArea
 	{
 		public:
-			ChannelArea( QWidget * parent, MixerView * mv ) :
-				QScrollArea( parent ), m_mv( mv ) {}
+			ChannelArea(QWidget * parent, MixerView * mv) :
+				QScrollArea(parent), m_mv(mv) {}
 			~ChannelArea() override = default;
-			void keyPressEvent( QKeyEvent * e ) override
+			void keyPressEvent(QKeyEvent * e) override
 			{
-				m_mv->keyPressEvent( e );
+				m_mv->keyPressEvent(e);
 			}
 		private:
 			MixerView * m_mv;
 	};
-	channelArea = new ChannelArea( this, this );
-	channelArea->setWidget( m_channelAreaWidget );
-	channelArea->setVerticalScrollBarPolicy( Qt::ScrollBarAlwaysOff );
-	channelArea->setFrameStyle( QFrame::NoFrame );
-	channelArea->setMinimumWidth( mixerLineSize.width() * 6 );
-	channelArea->setFixedHeight( mixerLineSize.height() +
-			style()->pixelMetric( QStyle::PM_ScrollBarExtent ) );
-	ml->addWidget( channelArea, 1, Qt::AlignTop );
+	channelArea = new ChannelArea(this, this);
+	channelArea->setWidget(m_channelAreaWidget);
+	channelArea->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+	channelArea->setFrameStyle(QFrame::NoFrame);
+	channelArea->setMinimumWidth(mixerChannelSize.width() * 6);
+	channelArea->setFixedHeight(mixerChannelSize.height() +
+			style()->pixelMetric(QStyle::PM_ScrollBarExtent));
```

Comment:
Not sure how others feel about it, but to me I'd prefer to have the operator on the line split, like this:
```suggestion
	channelArea->setFixedHeight(mixerChannelSize.height()
			+ style()->pixelMetric(QStyle::PM_ScrollBarExtent));
```

Reply:
Fixed with commit 92878ce4030 by using an intermediate variable so that all can go in one line.